src/OSPSuite.Core/Domain/Data/SimulationResults.cs

```diff
@@ -18,7 +18,9 @@ public class SimulationResults : IEnumerable<IndividualResults>
 
       public virtual QuantityValues Time { get; set; }
 
-      //We need to add the objects in a thread safe manner to the list that is intrinsicly not thread safe
+      public virtual IndividualResults[] IndividualResultsAsArray() => AllIndividualResults?.ToArray();
```

Comment:
R does not work with other more advanced strucutres. We need to export as Array. Using ToArray does not work also because it's generic